bitnami/harbor/values.yaml

```diff
@@ -275,58 +391,80 @@ persistence:
   ##
   resourcePolicy: 'keep'
   persistentVolumeClaim:
-    ## @param persistence.persistentVolumeClaim.registry.existingClaim Use the existing PVC which must be created manually before bound, and specify the `subPath` if the PVC is shared with other components
-    ## @param persistence.persistentVolumeClaim.registry.storageClass Specify the `storageClass` used to provision the volume. Or the default StorageClass will be used (the default). Set it to `-` to disable dynamic provisioning
+    ## @param persistence.persistentVolumeClaim.registry.existingClaim Name of an existing PVC to use
+    ## @param persistence.persistentVolumeClaim.registry.storageClass PVC Storage Class for Harbor Registry data volume
+    ## Note: The default StorageClass will be used if not defined. Set it to `-` to disable dynamic provisioning
     ## @param persistence.persistentVolumeClaim.registry.subPath The sub path used in the volume
-    ## @param persistence.persistentVolumeClaim.registry.accessMode The access mode of the volume
+    ## @param persistence.persistentVolumeClaim.registry.accessModes The access mode of the volume
```

Comment:
Putting it here because I cannot mark the correct line.

In the template I can see
```
persistence.imageChartStorage.s3.storageclass
```

but in the values.yaml I see

```
persistence.imageChartStorage.s3.storageClass
```